turte_graphics/turtle.js

```diff
@@ -0,0 +1,111 @@
+class Turtle {
+  constructor(x, y) {
```

Comment:
I might name these `startX` and `startY` or `initialX` and `initialY` as "documentation" to the future reader (I think self-explanatory variable names are better than comments which can get out of date very easily)

Reply:
done